docs/point_api_spec.md

```diff
@@ -0,0 +1,69 @@
+# Address Point API Spec
+
+## Purpose
+This document describes the Address Point Geocoding API, serving as a specification or "contract" for other users of this service.
+In particular, it describes the endpoints that this service exposes, as well as the structure and format of the data that is exchanged when using this service.
+
+### Protocols and data formats
+The Address Point Geocoding Service exposes data through HTTP, accepting and returing data in JSON format. 
+In particular, all requests need to specify their content type as appropriate:
+
+```
+Content-Type: application/json
+```
+
+Requests that don't specify their content type will be rejected
+
+### API Endpoints
+
+`GET  /status`
+
+Returns a message with current status and date
+
+```json
+{
+    "status": "OK",
+    "time": "Wed Feb 18 10:30:55 EST 2015"
+}
+```
+
+`POST /address/point`
```

Comment:
Why no `GET` for single point lookups?  I _get_ that we'll have to support `POST` for batch lookups due to the ~2000 character limit on URLs, but this seems pretty limiting for one-offs.  Its definitely easier for a developer checking out out our API for the first time to be able to play around with URLs in a browser, and get results back.  Its also seems like it would be very helpful to be able to share a search URL when someone has a question about the results.

In addition, caching is more straightforward with `GET`.  You [can cache `POST`s](http://stackoverflow.com/questions/626057/is-it-possible-to-cache-post-methods-in-http), but it requires some extra headers to be set, which folks are unlikely to set per request.

Something like:
>`GET /address/point?search=16410+N+AR+94+Hwy+Pea+Ridge+72751+AR`